ppa/archive/templates/archive/snippets/pagination.html

```diff
@@ -8,11 +8,11 @@
     <div class="page-controls">
             {# previous link at first (disabled if no previous)  #}
             {% if page_obj.has_previous %}
-            <a rel="prev" class="ui icon basic button" href="?{% querystring_replace page=page_obj.previous_page_number %}">
+            <a name="previous page" title="previous page" rel="prev" class="ui icon basic button" href="?{% querystring_replace page=page_obj.previous_page_number %}">
```

Comment:
does pa11y require both name and title?

Reply:
yeah...I think it's because the link doesn't have an ID? I remember it prompting me to either add an ID or name even _after_ I added title in some cases; this was one. might be something to do with creating a unique handle for the link for the screen reader?